packages/spicetify-creator/src/scripts.ts

```diff
@@ -36,14 +36,22 @@ const build = async (watch: boolean, minify: boolean, outDirectory?: string, inD
     fs.mkdirSync(outDirectory, { recursive: true });
   }
 
+  // Load PostCSS config or fallback to autoprefixer
+  let postCSSPlugins = [autoprefixer];
+  try {
+    const postcssrc = require('postcss-load-config');
```

Comment:
Why is `require` needed in the `try...catch` block? Can you move this to the top of the file like the rest of the modules?